common/transport/src/main/proto/grpc/block_worker.proto

```diff
@@ -232,6 +252,16 @@ message BlockStatus {
   optional bool retryable = 4;
 }
 
+message LoadFileFailure {
+  optional string alluxio_path = 1;
```

Comment:
this is actually not used... should we keep it?

Reply:
we can remove it if not used